test/attr/attr_objc_typealias_inside_protocol.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %empty-directory(%t)
+// RUN: %swift-target-frontend -enable-objc-interop -parse-as-library %s -o /dev/null -disable-objc-attr-requires-foundation-module -module-name reduced -emit-module -emit-module-path %t/reduced.swiftmodule
+@objc public protocol P {
+  typealias T = () -> Bool
```

Comment:
@jrose-apple might suggest a way to explicitly check for the presence or absence of `@objc` attribute, perhaps by printing the interface using swift-ide-test?

Reply:
Good point. It failed to build before, but you could also check this by copying the swift-ide-test invocation from attr_objc.swift. (Also, you got `%target-swift-frontend` backwards—please run tests locally too!)

Reply:
Ah sorry, I have fixed the test. Running the tests locally now!